.pre-commit-config.yaml

```diff
@@ -9,10 +9,7 @@ repos:
     hooks:
       - id: mypy
         args: [--strict]
-  - repo: https://github.com/psf/black
-    rev: 21.7b0
-    hooks:
-      - id: black
```

Comment:
I think you accidentally removed `black` in your patch?

Reply:
@mattyclarkson  the hook is duplicated in the files so i have deleted the second entry..

Reply:
ah, makes sense :)